config/samples/csharp_example_loadtest.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: csharp-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: csharp
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+    - language: csharp
+      clone:
+        repo: https://github.com/jtattermusch/grpc.git
+        # TODO(jtattermusch): we cannot use grpc/grpc master because currently C# qps_worker doesn't accept the --server_port arg
```

Comment:
We removed server ports in #97 refactor, they are completely scrubbed from the defaults in #99. It may be better to wait for one of those, so we can use the official repo here.

Reply:
I think this PR is mostly about adding support for csharp, example is just a part of it. Since the main part is done, I think it may be better merge this first and update it when 97 get merged. I am okay with both solution actually. :)